Cargo.toml

```diff
@@ -313,7 +313,7 @@ sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch
 substrate-prometheus-endpoint = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
 
 # Cli specific
-frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
+frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43", features = ["rocksdb"]}
```

Comment:
Enabling this feature is not really necessary as it is the default one. Happy to drop this here but I vote for rather being explicit.